shimming-toolbox/shimmingtoolbox/masking/mask_utils.py

```diff
@@ -228,3 +283,145 @@ def modify_binary_mask(mask, shape='sphere', size=3, operation='dilate'):
+
+    Args:
+        path_sct_binmask (str): Path to the binary mask created from the `sct_create_mask` function.
+        soft_width (int): Width of the soft zone (in pixels). Must be a multiple of 3. In this case, the soft zone is a gaussian blur and its
+                            width is defined by the number of pixels with an intensity larger than 0.5.
+    Returns:
+        sct_softmask (numpy.ndarray): soft mask created from the binary mask.
+    """
+    # Load the binary mask from a NIFTI file
+    nifti_file = nib.load(path_sct_binmask)
+    sct_binmask = nifti_file.get_fdata()
+
+    # Raise error if soft_width is not a multiple of 3
+    if soft_width % 3 != 0:
+        raise ValueError("soft_width must be a multiple of 3")
+
+    # Create a np.array soft mask
+    sct_softmask = np.array(sct_binmask, dtype=float)
+    previous_mask = np.array(sct_binmask, dtype=float)
+    for _ in range(soft_width // 3):
+        dilated_mask = binary_dilation(previous_mask, ball(3))
+        previous_mask = dilated_mask
+    blurred_mask = gaussian_filter(previous_mask.astype(float), soft_width / 2)
+    blurred_mask[blurred_mask < 0.1] = 0
+    sct_softmask = np.clip(blurred_mask + sct_binmask, 0, 1)
+
+    return sct_softmask
+
+
+def gaussian_sct_softmask(path_sct_binmask, path_sct_gaussmask):
+    """
+    Adapts the gaussian filter created by sct_create_mask to create a soft mask containing a gaussian blur from the
+    binary mask to the background, while keeping the binary mask.
+
+    Args:
+        path_sct_binmask (str): Path to the binary mask created from the `sct_create_mask` function.
+        path_sct_gaussmask (str): Path to the gaussian mask created from the `sct_create_mask` function.
+    Returns:
+        sct_softmask (numpy.ndarray): soft mask created from the SCT masks.
+    """
+    # Load the sct binary mask from a NIFTI file
+    nifti_file = nib.load(path_sct_binmask)
+    sct_binmask = nifti_file.get_fdata()
+
+    # Load the sct gaussian mask from a NIFTI file
+    nifti_file = nib.load(path_sct_gaussmask)
+    sct_gaussmask = nifti_file.get_fdata()
+
+    # Create a np.array soft mask
+    sct_gaussmask[sct_gaussmask < 0.1] = 0
+    sct_softmask = np.clip(sct_gaussmask + sct_binmask, 0, 1)
+
+    return sct_softmask
+
+
+def save_softmask(sct_softmask, path_sct_softmask, path_sct_binmask):
+    """
+    Save the soft mask to a NIFTI file
+
+    Args:
+        sct_softmask (numpy.ndarray): Soft mask to save.
+        path_sct_softmask (str): Path to save the soft mask.
+        path_sct_binmask (str): Path to the binary mask.
+    Returns:
+        nii_sct_softmask : NIFTI file containing the soft mask created from the binary mask.
+    """
+    nifti_file = nib.load(path_sct_binmask)
+
+    nii_sct_softmask = nib.Nifti1Image(sct_softmask, nifti_file.affine)
+    nii_sct_softmask.set_data_dtype(float)
+    nii_sct_softmask.to_filename(path_sct_softmask)
+
+    return nii_sct_softmask
```

Comment:
This is a generic function, sct should not be referenced here. Also, the docstring for the output is the type followed by the description, not the name of the variable.
```suggestion
def save_softmask(softmask, path_softmask, path_binmask):
    """
    Save the soft mask to a NIFTI file

    Args:
        softmask (numpy.ndarray): Soft mask to save.
        path_softmask (str): Path to save the soft mask.
        path_binmask (str): Path to the binary mask.
    Returns:
        nib.Nifti1Image : NIFTI file containing the soft mask created from the binary mask.
    """
    nifti_file = nib.load(path_binmask)

    nii_softmask = nib.Nifti1Image(softmask, nifti_file.affine)
    nii_softmask.set_data_dtype(float)
    nii_softmask.to_filename(path_softmask)

    return nii_softmask
```

Reply:
All references to SCT have been removed in commit [27e24ab](https://github.com/shimming-toolbox/shimming-toolbox/pull/581/commits/27e24abbb6500b4b38747bc407130516c4a349ec)